03/app.test.js

```diff
@@ -1 +1,71 @@
-import randomNumber from './app';
\ No newline at end of file
+import randomNumber from './app';
+
+it('randomNumber is a number', () => {
+    expect(typeof randomNumber(10, 1)).toBe('number');
+})
+
+it('parameter min must be less or equal as parameter max', () => {
+
+    randomNumber.min = 11;
+    randomNumber.max = 10;
+    if (randomNumber.min > randomNumber.max) {
+        randomNumber.min = randomNumber.max;
+    }
+
+    expect(randomNumber.min <= randomNumber.max).toBe(true);
+})
+
+it('result must be greater or equal as parameter min', () => {
+
+    randomNumber.min = 5;
+    randomNumber.max = 10;
+    randomNumber.result = 4;
+    if (randomNumber.result < randomNumber.min) {
+        randomNumber.result = randomNumber.min;
+    }
+
+    expect(randomNumber.result >= randomNumber.min).toBe(true);
+})
```

Comment:
Ponieważ mamy funkcje to w ten sposób nie sprawdzimy parametrów.
Parametry to zmienne lokalne, które nie są dostępne na zewnątrz.
Powinieneś raczej sprawdzić czy wylosowana liczba jest z odpowiedniego przedziału.